### ISSUE TITLE
BUG: Zone entities with a “shapeType” other than “Box” cause a glitches

### ISSUE THREAD

**Issue by AleziaKurdis** [vircadia/vircadia#172]
If we set the “shapeType” of a Zone entity to “cylinder” or “sphere”, when the user enters the cube of the zone but is still outside the cylinder or the sphere (let say he’s in the corner) then a glitch happens.

The Zone is momentarily applied to be immediately reverted. It behaves as if the evaluation of our position related to the sphere or cylinder was done after have been considered inside the zone.

Priority: Medium

**Comment by HifiExperiments** [comment]
@AleziaKurdis is this still a problem?  I've tried to reproduce it a few times since this was posted and never could.  I'm using a zone with a skybox color and everything else set to inherit.  does it happen with some particular property configuration?  from the code, I don't see how this would be possible

**Comment by AleziaKurdis** [comment]
1- Go there: http://metaverse.bashora.com/explore/serverlessWothal-UM.json
2- using create app, set radius to 10000 and find the Zone entity: "PLANET W ATMOSPHERE"
3- Rotate it to -135 deg on the X axis it to be in a corner outside the zone spheric shape

4- Now look the horizon, and click on the arrow of the X field to rotate it with a fraction of degree and observe the refresh.. you see momentarily the sky of that zone.
![image](https://user-images.githubusercontent.com/60075796/107463932-2dbd1980-6b2d-11eb-8ede-103bd235f560.png)

(Originally, there was a domain script doing that slow rotation, so the glitch was very visible. Sadly. I decommissioned this content, but you still can see the refresh issue rotating it manually.)
Sorry to can't provide you something better to reproduce it.

This is clearly not a high priority problem. But there is glitch.
There is certainly a workaround with the renderWithZones feature for this specific case (it wasn't existing when I created that)
So we can say that the priority just dropped from medium to low
